tests/component/test_write_properties.py

```diff
@@ -0,0 +1,126 @@
+"""Tests for validating write properties."""
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx.constants import WriteRelativeTo
+from nidaqmx.errors import DaqError, DAQmxErrors
+from nidaqmx.task import Task
+
+
+@pytest.fixture(scope="function")
+def write_task(any_x_series_device):
```

Comment:
We normally call this an AO task or an output task, not a write task.

Rename the test cases' given clauses accordingly, like test__ao_task__...

Reply:
Updated the method names.